### PR TITLE
Addition of omChain Network

### PR DESCRIPTION
**Pull request by nithronium**
addition of omChain to Chainlist

### CODE REVIEW

File: _data/chains/eip155-21816.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "symbol": "OML",
+    "decimals": 18
+  },
+  "infoURL": "https://omlira.com",
+  "shortName": "oml",
+  "chainId": 21816,
+  "networkId": 21816,
+  "explorers": [{
+    "name": "omChain Explorer",
+    "url": "https://explorer.omlira.com"
```
Comment (ligi) [inline _data/chains/eip155-21816.json:20]:
```suggestion
    "url": "https://explorer.omlira.com",
    "standard": "EIP3091"
```

Comment (Breadcorn) [reply _data/chains/eip155-21816.json]: bc1q9846l20qagljl85xhucszstqlw4tk2usxrwyk5

**Comment by amin1419** [comment]
Confirm

**Review by amin1419 — COMMENTED** [review]
Confirm

**Comment by ligi** [comment]
don't confirm - fix ;-)

**Comment by nithronium** [comment]
> don't confirm - fix ;-)

fixed the icon

**Pull request merged**